feature/apps/presentation/src/commonMain/kotlin/zed/rainxch/apps/presentation/starred/StarredPickerViewModel.kt

```
@@ -0,0 +1,197 @@
+                    }
+                    current.copy(
+                        candidates = updated.toImmutableList(),
+                        scanProgress = ++processed,
+                    )
+                }
+            } catch (e: RateLimitException) {
+                _state.update {
+                    it.copy(rateLimited = true, errorMessage = e.message)
+                }
+                return
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)